src/game.js

```diff
@@ -253,18 +271,30 @@
 
                 // update the timer
                 me.timer.update(time);
-
-                // clear the quadtree
-                me.collision.quadTree.clear();
-
-                // insert the world container (children) into the quadtree
-                me.collision.quadTree.insertContainer(api.world);
-
-                // update all objects (and pass the elapsed time since last frame)
-                isDirty = api.world.update(me.timer.getDelta()) || isDirty;
-
-                // update the camera/viewport
-                isDirty = api.viewport.update(me.timer.getDelta()) || isDirty;
+                
+                accumulator += me.timer.getDelta();
+                
+                updateDelta = (me.sys.interpolation) ? me.timer.getDelta() : stepSize;
+    
+                while (accumulator >= stepSize || me.sys.interpolation) {
```

Comment:
:laughing: You just need a lot of entities with a lot of AI stuff. It's actually not too difficult to get long update times. Especially on slow devices like my awful iPhone 4S ...